src/proteus/atmos_clim/agni.py

```diff
@@ -328,16 +361,16 @@ def RunAGNI(atmos, loops_total:int, dirs:dict, OPTIONS:dict, hf_row:dict):
                             conduct=False, convect=True, latent=True, sens_heat=True,
 
                             max_steps=130, max_runtime=900.0,
-                            conv_atol=1e-3, conv_rtol=1e-2,
+                            conv_atol=1e-3, conv_rtol=2e-2,
```

Comment:
Should these solver tolerances be hardcoded or rather go into the config file?

Reply:
It should do, yeah. I think it could go in the `[atmosphere]` section of the TOML configuration file once we have reformatted it in that way (issue #74).

Reply:
Or even in a template configuration file of some kind

Reply:
Ok, let's leave them here for now then.

Reply:
See https://github.com/FormingWorlds/PROTEUS/issues/162